modules/claims_api/lib/bgs_service/corporate_update_web_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class CorporateUpdateWebService < ClaimsApi::LocalBGS
+    def bean_name
+      'CorporateUpdateServiceBean'
+    end
+
+    def corporate_update
+      self
+    end
```

Comment:
What is the purpose of this method? I couldnʼt find any other examples of this pattern in the other services.

Reply:
I thought it was needed based on the way we are calling the service, but could be wrong. Let me take a look...

Reply:
Removed from latest push.